### PR TITLE
Add types for lightpick.

### PR DESCRIPTION
**Pull request by akwiatek**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldnt have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/lightpick/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
```
Comment (amcasey) [inline types/lightpick/tsconfig.json]: It looks like the underlying library is UMD, so you might want to have a look at this template: https://github.com/microsoft/TypeScript-Handbook/blob/master/pages/declaration%20files/templates/module-class.d.ts.md

File: types/lightpick/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,384 @@
+// Type definitions for Lightpick 1.3
+// Project: https://wakirin.github.io/Lightpick
+// Definitions by: Adam Kwiatek <https://github.com/akwiatek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import * as moment from 'moment';
```
Comment (amcasey) [inline types/lightpick/index.d.ts]: Generally speaking, you can't instantiate a symbol imported in this way.  Maybe you meant `import moment = require('moment');`?

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/lightpick/index.d.ts (hunk 2 above)
Comment (kwiatek-kainos) [reply types/lightpick/index.d.ts]: Done.

File: types/lightpick/tsconfig.json (hunk 1 above)
Comment (kwiatek-kainos) [reply types/lightpick/tsconfig.json]: Thanks. I've made adjustments according to that template and type definitions for Lodash and jQuery.

I'm not entirely sure though, why you put a comment against tsconfig.json . Should I change anything here? Configs for other UMD types look similar...

Comment (amcasey) [reply types/lightpick/tsconfig.json]: Sorry, I blanked.  I was thinking that the module type should be "umd", but that doesn't matter for .d.ts files.

**Review by amcasey — APPROVED** [review]
Thanks for the changes!

**Pull request merged**